indian-languages/README.md

```diff
@@ -0,0 +1,9 @@
+# Bharati Braille
+
+Most Indian languages use Bharati Braille a system developed in the 1950's to unify as far as possible the Braille codes for the various Indian languages and scripts.
+
+The documents here were supplied by Braille Council of India.
+
+`Braille chart Pg-1.jpg` and `Braille chart pg-2.jg` show a summary of the Bharati Braille system. For how it applies to individual languages refer to [World Braille Usage (3rd edition)](../world-braille-usage-third-edition.pdf).
+
+The documents in [`recommended-changes-2022`](recommended-changes-2022) directory show changes which are being considered, but not yet officially adopted, in 2022.
```

Comment:
I don't think we can link to a directory. It's better to link to the individual files, with a link text.

Reply:
OK I will do this. Also in regards to them being accessible how about if I type out the information contained in them? We could keep the images to validate the information.

Reply:
Sure, if you think it is worth the trouble? (I mean, since they're only "recommended changes" and not official yet.)